new-dti-website/components/team/MemberGroup.tsx

```diff
@@ -62,17 +64,18 @@ type MemberCardProps = {
 };
 
 const MemberCard: React.FC<MemberCardProps> = (props: MemberCardProps) => (
-  <Card
-    id="memberCard"
-    className={`w-fit md:p-3 md:pb-4 xs:p-2 xs:pb-3 h-fit justify-self-center ${
-      props.cardState ? 'opacity-70 hover:opacity-100' : 'opacity-100'
-    } ${
-      props.cardState === 0 && 'shadow-[0_4px_4px_0_#00000040]'
-    } hover:shadow-[0_4px_4px_0_#00000040] cursor-pointer`}
-    onClick={props.onClick}
-  >
-    <MemberSummary {...props} enlarged={false} />
-  </Card>
+  <button onClick={props.onClick}>
+    <Card
+      id="memberCard"
+      className={`w-fit md:p-3 md:pb-4 xs:p-2 xs:pb-3 h-fit justify-self-center ${
+        props.cardState ? 'opacity-70 hover:opacity-100' : 'opacity-100'
+      } ${
+        props.cardState === 0 && 'shadow-[0_4px_4px_0_#00000040]'
+      } hover:shadow-[0_4px_4px_0_#00000040] cursor-pointer`}
+    >
+      <MemberSummary {...props} enlarged={false} />
+    </Card>
+  </button>
```

Comment:
I can use `tab` to navigate between the different team member cards, but I cannot use my keyboard (`enter` or `space`) to open the card. Instead I have to use my mouse. The cards should be keyboard accessible